docs/components/rule.md

```diff
@@ -0,0 +1,16 @@
+# Rule
+
+The rule component evaluates Prometheus recording and alerting rules against random query nodes in its cluster. Rule results are written back to disk in the Prometheus 2.0 storage format. Rule nodes at the same time participate in the cluster themselves as source store nodes and upload their generated TSDB blocks to an object store.
+
+The data of each rule node can be labeled to satisfy the clusters labeling scheme. High-availability pairs can be run in parallel and should be distinguished by the designated replica label, just like regular Prometheus servers.
+
```

Comment:
Can we add note to make sure ruler will query deduplicated data (per https://github.com/improbable-eng/thanos/issues/137 discussion)